node-graph/gcore/src/ops.rs

```diff
@@ -30,18 +31,71 @@ where
 	first + second
 }
 
-pub struct MulParameterNode<Second> {
+// Subtract
+pub struct SubtractParameterNode<Second> {
+	second: Second,
+}
+
+#[node_macro::node_fn(SubtractParameterNode)]
+fn flat_map<U, T>(first: U, second: T) -> <U as Sub<T>>::Output
+where
+	U: Sub<T>,
+{
+	first - second
+}
+
+// Divide
+pub struct DivideParameterNode<Second> {
 	second: Second,
 }
 
-#[node_macro::node_fn(MulParameterNode)]
+#[node_macro::node_fn(DivideParameterNode)]
+fn flat_map<U, T>(first: U, second: T) -> <U as Div<T>>::Output
+where
+	U: Div<T>,
+{
+	first / second
+}
+
+// Multiply
+pub struct MultiplyParameterNode<Second> {
+	second: Second,
+}
+
+#[node_macro::node_fn(MultiplyParameterNode)]
 fn flat_map<U, T>(first: U, second: T) -> <U as Mul<T>>::Output
 where
 	U: Mul<T>,
 {
 	first * second
 }
 
+// Exponent
+pub struct ExponentParameterNode<Second> {
+	second: Second,
+}
+
+#[node_macro::node_fn(ExponentParameterNode)]
+fn exp<U, T>(first: U, second: T) -> <U as Pow<T>>::Output
+where
+	U: Pow<T>,
+{
+	first.pow(second)
+}
+
+// Modulo
+pub struct ModuloParameterNode<Second> {
+	second: Second,
+}
+
+#[node_macro::node_fn(ModuloParameterNode)]
+fn flat_map<U, T>(first: U, second: T) -> <U as Rem<T>>::Output
```

Comment:
```suggestion
fn mod<U, T>(first: U, second: T) -> <U as Rem<T>>::Output
```

Reply:
changed from "mod" to "modulo" as the build was failing